README.md

```diff
@@ -10,6 +10,8 @@
 <br/>
 </div>
 
+[SPV Wallet GitBook](https://bsvblockchain.gitbook.io/docs)
```

Comment:
IMHO this link should be introduced with some text, e.g. what it is and what an user can find there.